app/presenters/publishing_api/content_block_presenter.rb

```diff
@@ -0,0 +1,44 @@
+module PublishingApi
+  class ContentBlockPresenter
+    def initialize(schema_id:, content_id_alias:, edition:)
+      @schema_id = schema_id
+      @content_id_alias = content_id_alias
+      @edition = edition
+    end
+
+    def present
```

Comment:
Would it be worth naming this something to do with the usage? For example `ContentBlockPresenter.for_publishing_api` or `ContentBlockPresenter.as_content_store_compatible` or something along those lines?